validation-test/Evolution/test_rth.swift

```diff
@@ -13,7 +13,11 @@ let clientIsAfter = true
 
 let execPath = CommandLine.arguments.first!
 // FIXME: Don't hardcode "/" here.
```

Comment:
Do we still need the FIXME, then?

Reply:
I'm torn.  I really want a `Path` type so that we can deal with path separators (lets be real - the one true path separator is `:` from HFS 🤣).  I'm happy to remove the comment if you feel otherwise.

Reply:
Either way, honestly. @drodriguez did have the same thought.

Reply:
As you wish.